source/blood/src/fx.cpp

```diff
@@ -289,7 +289,7 @@ void fxSpawnBlood(spritetype *pSprite, int a2)
     }
 }
 
-void sub_746D4(spritetype *pSprite, int a2)
+void fxSpawnBloodPod(spritetype *pSprite, int a2)
```

Comment:
what's a blood pod?

Reply:
Well fxSpawnBloodPod() operates the same as fxSpawnBlood() but for green/red pod enemies.

Reply:
OOOOOH. Maybe fxSpawnPodBlood() would be better? Could just be me.

Reply:
I agree, fixed.